src/layout/questionnaire-list/components/questionnaire-list.jsx

```diff
@@ -34,17 +34,25 @@ class QuestionnaireList extends Component {
     super(props);
     this.state = {
       filter: '',
+      loaded: false,
     };
   }
 
   UNSAFE_componentWillMount() {
-    if (this.props.user && this.props.user.permission)
+    if (this.props.user && this.props.user.permission){
```

Comment:
As you are working on this method, this is maybe the best time to remove the UNSAFE-prefixed method. You should have a look to the react documentation in order to know what is the new syntax.

Reply:
Agreed. I will work on it.